hessian-lite/src/main/java/com/alibaba/com/caucho/hessian/io/JavaDeserializer.java

```diff
@@ -134,19 +134,19 @@ protected static Object getParamArg(Class cl) {
         else if (boolean.class.equals(cl))
             return Boolean.FALSE;
         else if (byte.class.equals(cl))
-            return new Byte((byte) 0);
+            return 0;
         else if (short.class.equals(cl))
-            return new Short((short) 0);
+            return 0;
         else if (char.class.equals(cl))
-            return new Character((char) 0);
+            return 0;
         else if (int.class.equals(cl))
-            return Integer.valueOf(0);
+            return 0;
```

Comment:
Good job, do you think we should merge these else statements?

Reply:
Do you mean change like this 
```java
protected static Object getParamArg(Class cl) {
        if (!cl.isPrimitive())
            return null;
        else if (boolean.class.equals(cl))
            return Boolean.FALSE;
        else if (byte.class.equals(cl) 
                || short.class.equals(cl) 
                || char.class.equals(cl) 
                || int.class.equals(cl))
            return 0;
        else if (long.class.equals(cl))
            return 0L;
        else if (float.class.equals(cl))
            return 0F;
        else if (double.class.equals(cl))
            return 0D;
        else
            throw new UnsupportedOperationException();
    }
```
It's ok with me to do so

Reply:
Never mind, let's keep is as it is.